java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/TextDocumentServiceImpl.java

```diff
@@ -1727,6 +1736,9 @@ public void changedUpdate(DocumentEvent e) {}
             }
             if (offset < 0) {
                 doc.putProperty("lsp-errors-" + keyPrefix, id2Errors);
+                doc.putProperty("lsp-errors-valid-" + keyPrefix, id2Errors);
```

Comment:
Are the keys invalidated e.g. after a modification ?

Reply:
Sure, see change at line 1500.